### PR TITLE
Some map stuff and some core stuff

### PR DESCRIPTION
**Pull request by ethteck**

### CODE REVIEW

File: tools/disasm_script.py [hunk 1]
```diff
@@ -38,10 +38,10 @@ def script_lib():
...
         with open(Path(path.dirname(__file__), "symbol_addrs.txt"), "r") as file:
             for line in file.readlines():
-                s = line.split(";")
-                name = s[0]
-                addr = s[1]
-                _script_lib[int(addr, 16)] = name
+                s = line.split("=")
+                name = s[0].strip()
+                addr = int(s[1].strip()[0:10], 0)
+                _script_lib[addr] = name
```
Comment (nanaian) [inline tools/disasm_script.py:44]: This'll conflict with #118

**Review by nanaian — APPROVED** [review]

**Review by pixel-stuck — CHANGES_REQUESTED** [review]

File: include/common.h [hunk 2]
```diff
@@ -10,5 +10,6 @@
 #include "enums.h"
 #include "si.h"
 #include "messages.h"
+#include "ld_addrs.h"
```
[the remark this answers is not in the document]
Comment (ethteck) [reply include/common.h:13]: your frown is my command

File: include/common_structs.h [hunk 3]
```diff
@@ -934,15 +936,23 @@ typedef struct GameStatus {
...
+    /* 0x160 */ Vec3s savedPos;
     /* 0x166 */ u8 saveSlot;
     /* 0x167 */ u8 loadType; /* (0 = from map, 1 = from main menu) */
     /* 0x168 */ s32 saveCount;
     /* 0x16C */ char unk_16C[12];
 } GameStatus; // size = 0x178
 
 typedef struct PartnerAnimations {
-    /* 0x00 */ char unk_00[36];
+    /* 0x00 */ UNK_PTR standard; // should be "default" but that's a keyword
```
[the remark this answers is not in the document]
Comment (ethteck) [reply include/common_structs.h:947]: I have no clue what they are..how confident are you?

File: include/variables.h [hunk 4]
```diff
@@ -86,6 +86,15 @@ extern PrintContext* gCurrentPrintContext;
 extern s32 D_802DB264;
 extern PrintContext* D_802DB268;
 
+// Animation related
```
[the remark this answers is not in the document]
Comment (ethteck) [reply include/variables.h:89]: not sure if just a joke, but this is me just grouping similar variables together so we can migrate them eventually

File: src/code_ef070_len_3400.c [hunk 5]
```diff
@@ -468,41 +465,235 @@ INCLUDE_ASM(s32, "code_ef070_len_3400", AdjustCam, ScriptInstance* script, s32 i
 
 INCLUDE_ASM(s32, "code_ef070_len_3400", ResetCam, ScriptInstance* script, s32 isInitialCall);
 
-INCLUDE_ASM(s32, "code_ef070_len_3400", func_802CCAC0);
+void func_802CCAC0(void) {
+    s32 i;
+
+    for (i = 0; i < MAX_ANIMATED_MODELS; i++) {
+        AnimatedModel* anim = (*gCurrentMeshAnimationListPtr)[i];
```
[the remark this answers is not in the document]
Comment (ethteck) [reply src/code_ef070_len_3400.c:472]: Eventually, yeah. but we don't have macros for any of the analogous lists yet, so I think it's best to hold off until we do them all at once

File: src/world/area_sbk/sbk_00/9292B0.c [hunk 6]
```diff
@@ -1,2 +1,70 @@
...
+    0x00310008, 0x00310008, 0x00310008, 0x00310008, 0x00310008, 0x00310008, 0x00310008, 0x00310008,
+    0x00000001, 0x00000000, 0x00000000, 0x00000000,
+};
+
+s32 M(npcGroupList_80240768)[] = {
+    0x00000001, M(npcGroup_80240388), 0x0A000001, 0x00000001, M(npcGroup_80240578), 0x0A010001, 0x00000000, 0x00000000,
+    0x00000000,
+};
+
+static s32 padding = 0;
```
[the remark this answers is not in the document]
Comment (ethteck) [reply src/world/area_sbk/sbk_00/9292B0.c:63]: It's much easier to figure out the splits we want to make later than to undo a wrong decision imo

File: src/world/script_api/7E3700.c [hunk 7]
```diff
@@ -33,19 +40,98 @@ INCLUDE_ASM(s32, "world/script_api/7E3700", GetGridIndexFromPos);
...
+    if (is_current_partner_flying()) {
+        partner->pos.y = playerStatus->position.y;
+    }
+
+    set_npc_yaw(partner, playerStatus2->targetYaw);
+    clear_partner_move_history(partner);
+    return ApiStatus_DONE2;
+}
+
+#ifdef NON_MATCHING
```
[the remark this answers is not in the document]
Comment (ethteck) [reply src/world/script_api/7E3700.c:82]: because I couldn't figure it out

[the remark this answers is not in the document]
Comment (pixel-stuck) [reply src/world/script_api/7E3700.c:82]: I think he means "what's wrong with the function" which we usually comment on non-matchings :P

[the remark this answers is not in the document]
Comment (ethteck) [reply src/world/script_api/7E3700.c:83]: ok so I looked back into this and got closer but wrote a comment :)

**Pull request merged**